### DISCUSSION TITLE
Removing bathymetry data at altitude

### DISCUSSION THREAD

**Discussion by dchristopherfennell** [pelicanmapping/osgearth#2317] [Support Q&A]
Glenn, 

You once gave me a program to help remove bathymetry data when at high altitudes above the Earth so we don't see the ocean floor mountains. This program works great, but as you said initially, osgEarth culls raw data so I get these bad side effects at the bottom of the screen.  

Do you have any magic that reduces these side effects?

Thanks,
DC

![Screenshot from 2023-08-01 09-55-53](https://github.com/gwaldron/osgearth/assets/7229222/627ac792-46cb-4e15-b266-0c6d64b2f834)

**Comment by gwaldron** [comment]
The only thing I can think of is to try calling `TerrainEngineNode::addModifyTileBoundingBoxCallback` and changing each tile's culling bounding box there. I am not sure at all that this will work :) and you'd need to experiment with the values. In general though, modifying verts in the shader will be invisible to OSG.
